packages/wonder-blocks-tokens/src/theme/semantic/semantic-color-thunderblocks.ts

```diff
@@ -558,7 +554,7 @@ export const semanticColor = mergeTheme(defaultSemanticColor, {
         rest: core.foreground.instructive.default,
         hover: core.foreground.instructive.strong,
         press: core.foreground.instructive.strong,
-        disabled: core.foreground.inverse.subtle,
+        disabled: core.foreground.disabled.subtle,
```

Comment:
`core.foreground.inverse.subtle` in TB was `gray_60`, ` core.foreground.disabled.subtle` is `gray_60` too!